docs/source/models/cause.rst

```diff
@@ -558,6 +558,85 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say that that 
+  year, every such case remitted after an average of 5 days:
+
+  .. math::
+
+    \frac{\text{1 case}}{\text{5 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
+
+  This calculation is straightfoward, as diarrheal diseases have a highly
+  consistent disease duration.
+
+  In contrast, consider diabetes. Say that there were 142,794 prevalent cases of
+  diabetes (both type I and type II) in Moldova amongst males in 2017, and of 
+  those 142,794 cases, 509 remitted in 2017. This gives us the following rate:
+
+  .. math::
+
+    \frac{\text{509 cases}}{\text{142,794 person-years}} =
+    \frac{\text{0.0036 cases}}{\text{1 person-year}}
+
+  It is important to note that this is a remission rate for diabetes at all
+  ages. In the context of the diarrheal diseases example, this makes it seem as 
+  if diabetes cases remit, on average, after :math:`\frac{1}{0.0036}\simeq 279` 
+  years--which clearly does a poor job of capturing the behavior of diabetes. 
+  This sort of description was only appropriate for diarrhea, as there is a 
+  uniform remission rate across all cases. With diabetes, remission is not 
+  guaranteed for type II, and doesn't occur for type I, so remission rate does 
+  *not* tell us the average duration that an individual will experience diabetes.
+
+For both of the above rates, we think of :math:`r_1=0.2` as the probability that 
```

Comment:
I was a bit confused about where r_1=0.2 came from (since the number 0.2 doesn't appear anywhere above), until I realized it was the remission rate for diarrhea measured in cases/person-day.

I think this gets at a technical mathematical point where we tend to be a bit sloppy in daily usage. Namely, the rates themselves don't represent probabilities, because they have physical dimensions of 1/time, whereas a probability needs to be dimensionless. Hence my confusion, because why can we use the daily remission rate as a probability but not the yearly rate (which clearly doesn't make sense because it's bigger than 1)?

Rather, to get a probability, we always have to multiply the rate by some unit of time (in this case 1 day for diarrhea, and 1 year for diabetes). Then we get a truly dimensionless number, z=(rate x time), and the probability of remission is given by 1-exp(-z). If z is **small enough**, then 1-exp(-z) is well-approximated by z, so we can use z=(rate x time) itself as an approximation for the probability.

A lot of this should eventually be explained on the survival analysis page, but I think whenever we want to convert rates to probabilities, we should always explicitly multiply by the appropriate time unit, and explicitly say that we can use this as an approximation to the true probability since (rate x time) is small enough.

Reply:
Ohh. Ok interesting; thank you for explaining that. I was a bit confused about how that worked but had been glazing over it. Updating.